CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ As a minor extension, we have adopted a slightly different versioning convention
 
 - Chain observers support the retrieval of the current Cardano chain point.
 
+- Deprecate `portable` feature of `mithril-stm` and `mithril-client`, BLST runtime check of intel ADX instruction set
```

Comment:
Maybe you could also say its has been removed from `mithril-common` and other crates?